lib/Loop/UvDriver.php

```diff
@@ -160,6 +160,13 @@ public static function isSupported(): bool {
         return \extension_loaded("uv");
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function now(): int {
+        return \uv_now($this->handle);
```

Comment:
`uv_now` is a monotonic timer and not related to the current real time.

Reply:
Right,that was my intention that will need to be clear through documentation, that `Loop::now()` is not necessarily wall time, but the loop time in milliseconds.